test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -71,42 +149,91 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 
         private void Repeater_Loaded(object sender, RoutedEventArgs e)
         {
-            sv.ChangeView((layout.ItemWidth + layout.Spacing) * 500, null, null, true);
+            sv.ChangeView(((layout.ItemWidth + layout.Spacing) * 500), null, null, true);
```

Comment:
>500 [](start = 65, length = 3)

you had some good pictures describing what the expressions are doing. would be nice to document them here.

Reply:
I can add that later when I finalize my iterations.

Reply:
I can add these in a later iteration after I finalize all of my proof-of-concepts.